compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/lowering_scalar_dispatch.mlir

```diff
@@ -35,9 +35,11 @@ hal.executable @scalar_dispatch {
 
 // CHECK-LABEL: func.func @scalar_dispatch()
 //  CHECK-SAME: translation_info = #iree_codegen.translation_info<pipeline = LLVMGPUBaseLowering workgroup_size = [1, 1, 1]>
-//       CHECK:   %[[SPAN0:.+]] = hal.interface.binding.subspan layout({{.+}}) binding(0)
-//       CHECK:   %[[SPAN1:.+]] = hal.interface.binding.subspan layout({{.+}}) binding(1)
-//       CHECK:   memref.load %[[SPAN0]][] : memref<i64, #hal.descriptor_type<storage_buffer>>
+//       CHECK:   %[[SPANBIND0:.+]] = hal.interface.binding.subspan layout({{.+}}) binding(0)
```

Comment:
I thought we were guarding the use of this on a flag. How did it get lowered to buffer ops here?

Reply:
The flag is on by default